### PR TITLE
Fix Go proxy

### PR DESCRIPTION
**Pull request by jld3103**
Fixes https://github.com/go-flutter-desktop/go-flutter/issues/392

### CODE REVIEW

File: cmd/bumpversion.go [hunk 1]
```diff
@@ -74,8 +74,9 @@ func upgradeGoFlutter(targetOS string, engineCachePath string) (err error) {
 	cmdGoGetU := exec.Command(build.GoBin(), "get", "-u", "github.com/go-flutter-desktop/go-flutter"+buildGoFlutterBranch)
 	cmdGoGetU.Dir = filepath.Join(wd, build.BuildPath)
 	cmdGoGetU.Env = append(os.Environ(),
-		"GOPROXY=direct", // github.com/golang/go/issues/32955 (allows '/' in branch name)
 		"GO111MODULE=on",
+		"GOPROXY=direct", // github.com/golang/go/issues/32955 (allows '/' in branch name)
+		"GOPRIVATE="+os.Getenv("GOPRIVATE"),
```
Comment (GeertJohan) [inline cmd/bumpversion.go:79]: I think this duplicates GOPRIVATE? On line 76 we already include the complete `os.Environ()`. This should already include GOPRIVATE. Same for the additions in the files below.

Having `GOPROXY=direct` does make sense, it overrides any value currently in the environment.

**Comment by jld3103** [comment]
@GeertJohan closing this PR, because we need to investigate more. Your comment shows that the bug @pchampio "discovered" isn't a bug, because it can't happen. https://github.com/go-flutter-desktop/go-flutter/issues/392#issuecomment-620738426

**Comment by GeertJohan** [comment]
Although the first file changed didnt contain os.Environ()... @jld3103

**Comment by jld3103** [comment]
Where it is used it contains it

**Comment by jinfagang** [comment]
SO you guys can access google site hard to reveal this problem? I'd like to help

**Comment by GeertJohan** [comment]
Yes we cannot reproduce the issue, so debugging is hard. @jinfagang

**Comment by jinfagang** [comment]
@GeertJohan I believe only people like me behind GFW can notice this. Simply google related go package can not access in China, can it switched to github? Or anythoer

**Comment by pchampio** [comment]
@jinfagang does https://github.com/go-flutter-desktop/go-flutter/issues/392#issuecomment-622950586 works?

Please report any issue/advancement in the issue tracker https://github.com/go-flutter-desktop/go-flutter/issues/392 and not this PR. It's easier for us.

**Pull request closed without merging**